tests/unit/drivers/rank/aggregate/test_aggregate_matches_rank_driver.py

```diff
@@ -67,38 +67,22 @@ def create_document_to_score_same_depth_level():
     # |  matches: (id: 4, parent_id: 30, score.value: 20, length: 2),
     # |  matches: (id: 5, parent_id: 30, score.value: 10, length: 1),
 
-    doc = jina_pb2.DocumentProto()
-    doc.id = str(1) * 16
+    doc = Document()
+    doc.id = 1
 
-    match2 = doc.matches.add()
-    match2.id = str(2) * 16
-    match2.parent_id = str(20) * 8
-    match2.length = 3
-    match2.score.ref_id = doc.id
-    match2.score.value = 30
-
-    match3 = doc.matches.add()
-    match3.id = str(3) * 16
-    match3.parent_id = str(20) * 8
-    match3.length = 4
-    match3.score.ref_id = doc.id
-    match3.score.value = 40
-
-    match4 = doc.matches.add()
-    match4.id = str(4) * 16
-    match4.parent_id = str(30) * 8
-    match4.length = 2
-    match4.score.ref_id = doc.id
-    match4.score.value = 20
-
-    match5 = doc.matches.add()
-    match5.id = str(4) * 16
-    match5.parent_id = str(30) * 8
-    match5.length = 1
-    match5.score.ref_id = doc.id
-    match5.score.value = 10
-
-    return Document(doc)
+    for match_id, parent_id, match_score, match_length in [
+        (2, 20, 30, 3),
+        (3, 20, 40, 4),
+        (4, 30, 20, 2),
+        (5, 30, 10, 1),
+    ]:
+        match = Document()
+        match.id = match_id
+        match.parent_id = parent_id
+        match.length = match_length
+        match.score = NamedScore(value=match_score, ref_id=doc.id)
+        doc.matches.append(match)
+    return doc
```

Comment:
parameterize this one

Reply:
Sure!

Reply:
I will fix it tomorrow

Reply:
You mean @pytest.fixture this function?

Reply:
@Yongxuanzhang sorry I didn't see it clearly, yes you can create a fixture, or leave it like this, looks good.

Reply:
Ok Thanks!